android/buildAndroidBOINC-CI.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+# This file is part of BOINC.
+# http://boinc.berkeley.edu
+# Copyright (C) 2018 University of California
+#
+# BOINC is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# BOINC is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with BOINC.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+wget -O /tmp/python3-argcomplete_0.8.1-1ubuntu2_all.deb http://ftp.ubuntu.com/ubuntu/ubuntu/pool/universe/p/python-argcomplete/python3-argcomplete_0.8.1-1ubuntu2_all.deb
+sudo dpkg -i /tmp/python3-argcomplete_0.8.1-1ubuntu2_all.deb
+
+sudo add-apt-repository ppa:ubuntu-desktop/ubuntu-make -y
+sudo apt-get update
+sudo apt-get --assume-yes install ubuntu-make
```

Comment:
Unfortunately, this cannot be done in before_script because first we need to download and manually install one of the dependencies, python3-argcomplete because when using default one from existing ubuntu repo it produces the error.

Reply:
I'll try to do smth with it

Reply:
Simply put the manual installation and the other sudo stuff in a separate script file and call this in the `before_install` section. This keeps all the system wide installations in one place and the CI scripts only contain the dependencies that are installed to the Cache.